applications/orochi-network-orosign-part1.md

```diff
@@ -0,0 +1,169 @@
+| **0b.** | Research    | Researching about Polkadot signature system and research their compatibility with MPC by which we can be fully compatible with Polkadot and its parachains |
+| **0c.** | Research    | Public article for every research we made, everything published under [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/)                        |
+| **1.**  | Research    | Researching about MPC in ECDSA (all supported signatures by Polkadot)                                                                                      |
+| **2.**  | Research    | Researching MPC for `secp256k1` and providing the document that describe how the MPC will be built and its security consideration                          |
+| **3.**  | Research    | Researching MPC for `ed25519` and providing the document that describe how the MPC will be built and its security consideration                            |
+| **4.**  | Research    | Researching MPC in `sr25519` and providing the document that describe how the MPC will be built and its security consideration                             |
+
+### Milestone 2
+
+- **Estimated Duration:** 1 month
+- **FTE:** 2
+- **Costs:** $7,000
+
+| Number  | Deliverable                           | Specification                                                                                                                                                                                                                                     |
+| ------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License                               | Apache 2.0                                                                                                                                                                                                                                        |
+| **0b.** | Documentation                         | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works.             |
+| **0c.** | Testing and Testing Guide             | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                                   |
+| **0d.** | Docker                                | We will publish an article that explains about Orosign's MPC wallet on curve `secp256k1`                                                                                                                                                          |
```

Comment:
I think you meant to change this milestone to "Article" instead of "Docker", can you fix this?